fees/thetis-market.ts

```diff
@@ -0,0 +1,56 @@
+import BigNumber from "bignumber.js";
+import { Adapter } from "../adapters/types";
+import fetchURL from "../utils/fetchURL";
+import { getUniqStartOfTodayTimestamp } from "../helpers/getUniSubgraphFees";
+import { CHAIN } from "../helpers/chains";
+
+const endpoint = "https://api.thetis.market/indexer/v1/stats/";
+
+const fetch = async (timestamp: number) => {
+  const startTime = getUniqStartOfTodayTimestamp(new Date(timestamp * 1000));
+  const endTime = startTime + 86400;
+
+  const feeRes = await fetchURL(
+    `${endpoint}fee-daily?startTime=${startTime}&endTime=${endTime}`
+  );
+
+  if (feeRes.length) {
+    const dailyFees = new BigNumber(feeRes[0].total).dividedBy(1e18);
+    const totalFees = new BigNumber(feeRes[0].cumulative).dividedBy(1e18);
+
+    return {
+      totalFees: totalFees.toString(),
+      totalSupplySideRevenue: totalFees.times(80).dividedBy(100).toString(),
+      dailyFees: dailyFees.toString(),
+      dailySupplySideRevenue: dailyFees.times(80).dividedBy(100).toString(),
```

Comment:
can also track dailyRevenue as `fees * 0.2`? Also, it is in number format, no need to do toString, not sure, maybe BigNumber is also a valid response format.

Reply:
We have updated, revenue = 80% fees

Reply:
no, the update has a bug, old code is correct: 
![image](https://github.com/user-attachments/assets/2871350d-26af-41a4-9a05-abc04528a128)
https://docs.llama.fi/list-your-project/other-dashboards/dimensions

Reply:
Yes, I have reverted the code.